llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/llama_index/vector_stores/azureaisearch/base.py

```diff
@@ -813,7 +819,7 @@ def _create_index_document(self, node: BaseNode) -> Dict[str, Any]:
         node_metadata = node_to_metadata_dict(
             node,
             remove_text=True,
-            flat_metadata=self.flat_metadata,
+            flat_metadata=False,
```

Comment:
We should still use self.flat_metadata, but just change it to false on the class attribute?

Reply:
@EmanuelCampos was this resolved?

Reply:
@nerdai yes!

Reply:
thanks! :)